steward/src/cellars/adaptors/balancer_pool.rs

```diff
@@ -132,8 +134,15 @@ fn convert_single_swap(swaps: Vec<SingleSwap>) -> Result<Vec<AbiSingleSwap>, Err
     swaps
         .into_iter()
         .map(|s| {
+            let pool_id = hex::decode(s.pool_id).map_err(|e|
```

Comment:
Are there any checks we'd want to make on the incoming pool ID value/format before we decode it?